enterprise/frontend/src/embedding-sdk/components/public/CollectionBrowser/CollectionBrowser.unit.spec.tsx

```diff
@@ -37,7 +38,10 @@ describe("CollectionBrowser", () => {
   it("should render", async () => {
     await setup();
 
-    expect(screen.getByText("Type")).toBeInTheDocument();
+    await waitFor(() => {
```

Comment:
Things need more ticks to update now, so sometimes we have to explicitly wait for UI to be updated in unit tests.